spec/support/shipment_helper.rb

```diff
@@ -8,6 +8,10 @@ def create_shipment(options = {})
       end
     end
 
+    def complete_order(options = {})
+      FactoryGirl.create(:order, options.merge(state: 'complete'))
+    end
+
```

Comment:
The name is inappropriate. Do a Find in your browser for `complete_order` and you will see how many matches you get that are not calling this helper.

It doesn't look like a great helper, so you might as well just add it as a private function in the test where you use it.